src/Aliases.ts

```diff
@@ -14,6 +14,8 @@ export default class Aliases {
     }
 
     private static importAliases(shellName: string = process.env.SHELL): void {
+        if (process.platform === 'win32') return;
```

Comment:
Have you noticed my comment about an enum? https://github.com/black-screen/black-screen/pull/129#discussion_r39341819